libert_util/src/util.c

```diff
@@ -3134,6 +3134,28 @@ bool util_is_first_day_in_month_utc( time_t t) {
 }
 
 
+/*
+  Expects date in the order YYYY-MM-DD.
+*/
+
+bool util_sscanf_isodate(const char * date_token , time_t * t) {
+  int day   , month , year;
+
+  if (t)
+    *t = -1;
+
+  if (sscanf(date_token , "%d-%d-%d" , &year , &month , &day) == 3) {
+    if (t)
+      *t = util_make_date_utc(day , month , year );
+    return true;
+  }
+
+  return false;
+}
```

Comment:
I must admit that even after reading the implementation, I don't really know what `util_sscanf_isodate` does.

Could you add a small comment somewhere about what the semantic is?